### PR TITLE
modules: fix documentation

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
modules: fix documentation

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
modules

### CODE REVIEW

File: plugins/modules/postgresql_pg_hba.py [hunk 1]
```diff
@@ -18,7 +18,8 @@
 description:
    - The fundamental function of the module is to create, or delete lines in pg_hba files.
    - The lines in the file should be in a typical pg_hba form and lines should be unique per key (type, databases, users, source).
-     If they are not unique and the SID is 'the one to change', only one for C(state=present) or none for C(state=absent) of the SID's will remain.
+     If they are not unique and the SID is 'the one to change', only one for C(state=present) or
```
[the remark this answers is not in the document]
Comment (Andersson007) [reply plugins/modules/postgresql_pg_hba.py]: Good spot, committed! my automation doesn't watch this:) PTAL

File: plugins/modules/postgresql_pg_hba.py [hunk 2]
```diff
@@ -18,7 +18,8 @@
 description:
    - The fundamental function of the module is to create, or delete lines in pg_hba files.
    - The lines in the file should be in a typical pg_hba form and lines should be unique per key (type, databases, users, source).
-     If they are not unique and the SID is 'the one to change', only one for C(state=present) or none for C(state=absent) of the SID's will remain.
+     If they are not unique and the SID is 'the one to change', only one for I(state=present) or
+     none for C(state=absent) of the SID's will remain.
```
Comment (felixfontein) [inline plugins/modules/postgresql_pg_hba.py:22]:
```suggestion
     none for I(state=absent) of the SID's will remain.
```

**Review by tcraxs — APPROVED** [review]

**Comment by tcraxs** [comment]
LGTM

**Review by felixfontein — APPROVED** [review]

**Comment by Andersson007** [comment]
@tcraxs @felixfontein thanks for reviewing!

**Pull request merged**